src/components/SliderBox/components/SliderArrow/SliderArrow.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import ArrowForwardIosIcon from '@mui/icons-material/ArrowForwardIos';
+import ArrowBackIosIcon from '@mui/icons-material/ArrowBackIos';
+import './SliderArrow.scss';
+
+const enum ArrowDirection {
+  Previous = 'previous',
+  Next = 'next',
+}
```

Comment:
move to separete file and all letters should be capital

Reply:
fixed , moved to types